e2e-tests/browser/rfc019_selfsigned/config/node/nuts.yaml

```diff
@@ -6,4 +6,4 @@ http:
     address: :8081
 auth:
   contractvalidators:
-    - selfsigned
+    - employeeid
```

Comment:
why did this change?

Reply:
test just happend to work because of koanf bug, default is `[irma,dummy,employeeid]` this was overwitten to `[selfsigned,dummy,employeeid]`. Fixed koanf bug means its nog `[selfsigned]` which does not exist